pysoa/test/stub_service.py

```diff
@@ -7,16 +7,19 @@
     ServiceHandler,
 )
 from pysoa.client.settings import ClientSettings
-from pysoa.common.types import Error
+from pysoa.common.metrics import NoOpMetricsRecorder
 from pysoa.common.transport.local import LocalClientTransport
+from pysoa.common.types import Error
 from pysoa.server import Server
 from pysoa.server.action import (
     Action,
     ActionError,
 )
 
 
-def _make_stub_action(action_name, body={}, errors=[]):
+def _make_stub_action(action_name, body=None, errors=None):
+    body = body or {}
+    errors = errors or {}
```

Comment:
Wowzers! How did this bug never bite us before?? 😬

Reply:
It never caused any problems because nothing mutates those values. But this is more correct for sure.